src/test/java/net/floodlightcontroller/forwarding/ForwardingTestSuit.java

```diff
@@ -0,0 +1,21 @@
+package net.floodlightcontroller.forwarding;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+/**
+ * This test class bundles all Forwrding/Routing tests and run them all at once
+ *
+ * @author Qing Wang (qw@g.clemson.edu) at 4/20/18
+ */
+@RunWith(Suite.class)
+
+@Suite.SuiteClasses({
+        L3RoutingTest.class,
+        ForwardingTest.class
+})
+
+public class ForwardingTestSuit {
```

Comment:
1. Floodlight already has a test runner, right? Is there any reason for making another one?
2. I think you accidentally left the 'e' off of the word "Suite"

Reply:
You're right, it is not necessary to have this test suit. I was trying to use it to bundle forwarding test with routing test so I can run both at a same time. Already removed it.